tests/functions/module_web.test.ts

```diff
@@ -0,0 +1,28 @@
+import { evaluateCellText } from "../test_helpers/helpers";
+
+describe("HYPERLINK formula", () => {
+  test("The evaluated result is the link", () => {
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com", "Odoo")' })).toBe(
+      "Odoo"
+    );
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com")' })).toBe(
+      "https://www.odoo.com"
+    );
+    expect(evaluateCellText("A1", { A1: "=HYPERLINK(A2)", A2: "https://www.odoo.com" })).toBe(
+      "https://www.odoo.com"
+    );
+    expect(
+      evaluateCellText("A1", { A1: '=HYPERLINK(A2, "Odoo")', A2: "https://www.odoo.com" })
+    ).toBe("Odoo");
+    expect(
+      evaluateCellText("A1", { A1: "=HYPERLINK(A2, A3)", A2: "https://www.odoo.com", A3: "Odoo" })
+    ).toBe("Odoo");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("invalidUrl")' })).toBe("invalidUrl");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("")' })).toBe("[]()");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK(" ")' })).toBe(" ");
+    expect(evaluateCellText("A1", { A1: "=HYPERLINK(2)" })).toBe("2");
```

Comment:
actually, you should use `evaluateCell` instead of `evaluateCellText` to assert the value is the *number* 2, not the the string "2". 
Same for every assertion, except maybe the date where both should be checked (the value which is a number and the text value which should be formatted as a date)

Sorry to have misguided you